### PR TITLE
feat(route): 创业邦资讯

### PR DESCRIPTION
**Pull request by nczitzk**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/cyzone
/cyzone/author/1225562
/cyzone/label/创业邦周报
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
  - [ ] EN / 英文文档
  - [x] CN / 中文文档
- [x] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Please rebase your branch

File: lib/v2/cyzone/author.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        author_id: id,
+    });
+
+    ctx.state.data = Object.assign(
+        {},
+        {
+            item: items,
+        },
+        await getInfo(currentUrl, ctx.cache.tryGet)
+    );
```
Comment (TonyRL) [inline lib/v2/cyzone/author.js:20]:
```suggestion
    ctx.state.data = {
        ...{
            item: items,
        },
        ...(await getInfo(currentUrl, ctx.cache.tryGet)),
    };
```

File: lib/v2/cyzone/label.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        tag: name,
+    });
+
+    ctx.state.data = Object.assign(
+        {},
+        {
+            item: items,
+        },
+        await getInfo(currentUrl, ctx.cache.tryGet)
+    );
```
Comment (TonyRL) [inline lib/v2/cyzone/label.js:20]:
```suggestion
    ctx.state.data = {
        ...{
            item: items,
        },
        ...(await getInfo(currentUrl, ctx.cache.tryGet)),
    };
```

File: lib/v2/cyzone/radar.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            },
+            {
+                title: '作者',
+                docs: 'https://docs.rsshub.app/new-media.html#chuang-ye-bang-zuo-zhe',
+                source: ['/author/:id', '/'],
+                target: '/cyzone/author/:id',
+            },
+            {
+                title: '标签',
+                docs: 'https://docs.rsshub.app/new-media.html#chuang-ye-bang-biao-qian',
```
Comment (TonyRL) [inline lib/v2/cyzone/radar.js:19]:
```suggestion
                docs: 'https://docs.rsshub.app/routes/new-media#chuang-ye-bang-zi-xun',
                source: ['/channel/:id', '/'],
                target: '/cyzone/:id',
            },
            {
                title: '作者',
                docs: 'https://docs.rsshub.app/routes/new-media#chuang-ye-bang-zuo-zhe',
                source: ['/author/:id', '/'],
                target: '/cyzone/author/:id',
            },
            {
                title: '标签',
                docs: 'https://docs.rsshub.app/routes/new-media#chuang-ye-bang-biao-qian',
```

File: lib/v2/cyzone/author.js (hunk 1 above)
Comment (nczitzk) [reply lib/v2/cyzone/author.js:20]: 👌

Comment (nczitzk) [reply lib/v2/cyzone/author.js:20]: 👌

File: lib/v2/cyzone/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+              }
+    );
+
+    ctx.state.data = Object.assign(
+        {},
+        {
+            item: items,
+        },
+        await getInfo(currentUrl, ctx.cache.tryGet)
+    );
```
Comment (TonyRL) [inline lib/v2/cyzone/index.js:27]:
```suggestion
    ctx.state.data = {
        ...{
            item: items,
        },
        ...(await getInfo(currentUrl, ctx.cache.tryGet)),
    };
```

**Pull request merged**